modules/nickserv.cpp

```diff
@@ -84,7 +84,7 @@ class CNickServ : public CModule {
 		}
 
 		if (GetNV("IdentifyCmd").empty()) {
-			SetNV("IdentifyCmd", "PRIVMSG NickServ :IDENTIFY {password}");
+			SetNV("IdentifyCmd", "NICKSERV :IDENTIFY {password}");
```

Comment:
I testedx on freenode and it works with both with and without colon. I am not sure if it should be there, but I added it just in case, but I can remove it.